E2E-TESTING.md

```diff
@@ -64,6 +64,39 @@ driver.
 
 ```
 
+### Finding elements
+
+For inspecting components labels and ids you could use `Accessibility Inspector` available from Xcode's Developer tools.
+
+- Text elements could be found by name
+```
+ <Text>{'My name'}</Text> // driver.elementByName('My name')
+```
+- for other components `accessibilityLabel` could be defined (use [BEM](http://getbem.com/introduction/) for the labels)
+```
+<Menu>
+    <MyComponent accessibilityLabel="menu__my-component" />
+    { /* elementById('my-component') */ }
```

Comment:
Should be `elementById('menu__my-component')`. And nicer ID would be `menu--my-component`